tests/sanity/tests/model/common-page.ts

```diff
@@ -5,7 +5,7 @@ export class CommonPage {
     if (name !== 'first') {
       await page.locator('div.selectPopup input').fill(name.split(' ')[0])
     }
-    await page.locator('div.selectPopup div.list-item:first-child').click()
+    await page.locator('div.selectPopup div.list-item:first-child').click({ delay: 500 })
```

Comment:
Could we avoid delays?

Reply:
will try to remove it. 
With it was more stable for cases when filling several dropdowns and selectors